frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -98,6 +99,8 @@ public final class EnhancedGraphTable extends BaseEnhancedGraphTable
       DataProvider,
       Disposable {
 
+  private static final AtomicBoolean ENQUEUING_UPDATES_DISABLED = new AtomicBoolean(false);
```

Comment:
I'd say `..._ENABLED`, easier to reason about (`..._DISABLED` is kinda-negation)